src/routes/api/userRoute.js

```diff
@@ -127,6 +127,28 @@ const { isImage } = customValidator;
 *       500:
 *         description: Internal server error
 */
+
+/**
+ * @swagger
+ *
+ * /users/notification:
+ *    get:
+ *      summary: Available users
```

Comment:
Kindly review the message, you wrote as a summary. I think it might be `Get all available notification`